crates/router/src/db/payment_attempt.rs

```diff
@@ -545,6 +556,10 @@ mod storage {
                 }
 
                 enums::MerchantStorageScheme::RedisKv => {
+                    /*
+                     * BREAKING CHANGE:
```

Comment:
@Abhicodes-crypto, once this function supports multiple attempt do the necessary changes instructed in the BREAKING_CHANGE comments